### PR TITLE
Struct mech/beam lumped mass rot factor

### PR DESCRIPTION
**Pull request by KlausBSautter**
lumped mass matrix beam
1.) added lumped mass matrix for 2D beam
2.) added lumped rot coef

for 2nd:
as you can see [here](http://kis.tu.kielce.pl/mo/COLORADO_FEM/colorado/IFEM.Ch31.pdf) the rotational parts of a lumped mass matrix are not exactly defined. So far I have never needed another factor as 0. I just stumbled over an example today where this factor is actually needed. So this PR includes this possibility

### CODE REVIEW

**Comment by loumalouomega** [comment]
Huuum, this lumped factor should depend on the geometry?

**Comment by KlausBSautter** [comment]
@loumalouomega the geomtry is always a 2-noded line in this case

**Comment by loumalouomega** [comment]
> @loumalouomega the geomtry is always a 2-noded line in this case

I see

File: applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_2D2N.cpp [hunk 1]
```diff
@@ -153,38 +153,64 @@ void CrBeamElement2D2N::CalculateMassMatrix(MatrixType &rMassMatrix,
...
+    this->GlobalizeMatrix(rMassMatrix);
+  }
+
+  else
+  {
+    const double lumped_mass = A * L * rho;
+    double alpha = 0.00;
+
+    if (this->GetProperties().Has(LUMPED_MASS_ROTATION_COEFFICIENT)) {
+      alpha = GetProperties()[LUMPED_MASS_ROTATION_COEFFICIENT];
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_2D2N.cpp:204]: can you please also add the reference here?

**Review by philbucher — COMMENTED** [review]
you have to remove the tetgen module

**Comment by KlausBSautter** [comment]
@philbucher how to remove the tetgen module?

**Comment by philbucher** [comment]
you have to delete it and commit this

**Comment by KlausBSautter** [comment]
ahhhhh :D

**Comment by KlausBSautter** [comment]
> Kratos_Main/applications/DelaunayMeshingApplication/external_modules/tetgen 

?
@philbucher

**Comment by loumalouomega** [comment]
imagen
There is no escape

**Review by philbucher — APPROVED** [review]

**Pull request merged**